cassandane/Cassandane/Unit/RunnerXML.pm

```diff
@@ -38,46 +38,45 @@
 #
 
 package Cassandane::Unit::RunnerXML;
+use strict;
+use warnings;
+use vars qw($VERSION);
 
 use XML::Generator;
 use Time::HiRes qw(time);
 use Sys::Hostname;
 use POSIX qw(strftime);
 
-use strict;
-use warnings;
-use vars qw($VERSION);
-# XXX should this inherit from our own Cassandane::Unit::Runner?
-use base qw(Test::Unit::Runner);
+use lib '.';
```

Comment:
Why is this being added?  Feels like a code smell.  The contents of @INC should probably be correct by now, unless I don't know how this works — which is possible!

Reply:
Cassandane expects to run out of its own source tree, which means none of its modules are installed to anywhere `@INC` looks, so all of Cassandane needs to `use lib '.';` before the use lines for its own stuff.  See d03d49befde7

RunnerXML didn't already have this line because it was an oddball that didn't use any Cassandane modules, but now it does so it does.

Reply:
This (`use lib '.'`) is the kind of thing I'd expect in the program that uses the modules, not the modules themselves.  For example, `testrunner.pl`.  Then you could delete it from many other places.